plugin/etcd/README.md

```diff
@@ -213,4 +222,4 @@ If you query the zone name for `TXT` now, you will get the following response:
 ~~~ sh
 % dig +short skydns.local TXT @localhost
 "this is a random text message."
-~~~
+~~~
```

Comment:
what's this change? Fixing trailing whitespace?

Reply:
Yep looks like it